Library/include/CSP/Multiplayer/Components/AnimatedModelSpaceComponent.h

```diff
@@ -140,6 +141,22 @@ class CSP_API AnimatedModelSpaceComponent : public ComponentBase,
 	/// @return The index of the currently active animation.
 	void SetAnimationIndex(int64_t Value);
 
+	/// @brief Gets the material overrides of this component.
+	/// Should be in the format:
+	/// Key = Path to the model
+	/// Value = The material asset id
```

Comment:
I'm not sure we should refer to materials as "assets".  The fact that they are stored as assets isn't important (they could be stored anywhere).  I would personally prefer just saying **material id**.

Reply:
I will also update the naming on the Material getters